tests/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cosmos/cosmos-sdk/tests
 
-go 1.21
+go 1.22
+
+toolchain go1.22.0
```

Comment:
Same Q here.

Reply:
Same answer, auto-generated with the `go mod tidy` script